core/src/stored/status.cc

```diff
@@ -994,6 +1003,19 @@ static void sendit(PoolMem& msg, int len, StatusPacket* sp)
   }
 }
 
+static void sendit(const std::string& msg, StatusPacket* sp)
```

Comment:
This general function is same for all three daemon. Could move it somewhere into libbareos.

Reply:
Point taken. I should probably refactor the whole `sendit()`-family somewhere into the library.